src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -91,7 +91,14 @@ public void handle(Endpoint<?> endpoint) {
                         }
                     });
 
-                    this.subscribe(false);
+                    try {
+                        this.subscribe(false);
+                    } catch (IllegalArgumentException e) {
+                        httpServerRequest.response().setStatusCode(ErrorCodeEnum.SUBSCRIPTION_EXCLUSIVE.getValue())
```

Comment:
are you sure that for this kind of scenario, an exception is raised by the `subscribe` method?
You are catching the `IllegalArgumentException` you added before but are we sure that the subscribe fails just for this reason and catching the exception is the way to go?

Reply:
No, I am not sure with this at all. Sadly I did not find a way how to test this path and implement it correctly. This is "scratch" how it could be but we need test definitely.

Reply:
So I would say, let's remove it from this PR and will open an issue for that one.
Remove even raising the `IllegalArgumentException` above.